src/dialog-editor/components/modal-field-template/drop-down-list.html

```diff
@@ -114,14 +114,25 @@
 <div ng-if="vm.modalTabIsSet('options') && vm.modalData.dynamic">
   <form class="form-horizontal">
     <div ng-show="!vm.treeOptions.show">
-      <div pf-form-group pf-label="{{'Entry Point'|translate}}">
-        <div class="input-group">
-          <input type="text" class="form-control" ng-value="vm.showFullyQualifiedName(vm.modalData.resource_action)" disabled>
+      <div pf-form-group pf-label="Automation Type" class="entry_point_selector_types">
+        <select class="form-control" miq-select ng-model="vm.modalData.automation_type" ng-change="vm.treeOptions.onAutomationTypeChange()">
+          <option value="{{vm.treeOptions.automationTypes.automate}}" translate>Embedded Automate</option>
+          <option value="{{vm.treeOptions.automationTypes.workflow}}" translate>Embedded Workflows</option>  
```

Comment:
We probably should only show this if the prototype flag is enabled.  This can be found via API.